CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased](https://github.com/fivetran/go-fivetran/compare/v0.2.1...HEAD)
 
+## [0.2.2](https://github.com/fivetran/go-fivetran/compare/v0.2.1...v0.2.2) - 2021-09-22
+
+## Fixed
+
+- `DestinationConfigRequest.SecretKey` missing field secret_key added
```

Comment:
maybe just 

- `DestinationConfigRequest.SecretKey` missing field added

? We're already mentioning the field.